sdk/metricsadvisor/Azure.AI.MetricsAdvisor/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - `DimensionKey` now implements the `IEnumerable<KeyValuePair<string, string>>` interface. Dimension columns can now be enumerated.
 - Added method `Contains` to `DimensionKey` to check whether or not a dimension column is present.
 - Added a property setter to `MetricSeriesGroupDetectionCondition.SeriesGroupKey` and to `MetricSingleSeriesDetectionCondition.SeriesKey`.
+- Added property `DimensionsToSplitAlert` to `AnomalyAlertConfiguration` to allow splitting an alert into multiple ones.
```

Comment:
how common is the usage for this property? just wondering if there should be a sample showing it

Reply:
I think it's not that uncommon. This is used when customers have different people to handle different types of incidents, so splitting an incident into multiple alerts makes it easier to forward them to the right people. It's a property that's very simple to use, so I think it's worth adding it to samples.

I'll go through all MA samples today, so I expect to add it when going through the Alert Config ones.